plugins/hybrid_analysis/plugin.spec.yaml

```diff
@@ -1,145 +1,1109 @@
-  - Hybrid Analysis API v1
+  - Hybrid Analysis API v2
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/hybrid_analysis
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   vendor_url: https://www.hybrid-analysis.com/
 tags:
-- malware
-- sandbox
-- malware analysis
+  - malware
+  - sandbox
+  - malware analysis
 hub_tags:
-  use_cases: [threat_detection_and_response]
-  keywords: [malware_analysis]
-  features: []
+  use_cases: [ threat_detection_and_response ]
+  keywords: [ malware_analysis ]
+  features: [ ]
 types:
-  report:
-    environment_id:
+  certificates:
+    issuer:
+      title: Issuer
+      type: string
+      description: Issuer
+      required: false
+    md5:
+      title: MD5
+      type: string
+      description: MD5
+      required: false
+    owner:
+      title: Owner
+      type: string
+      description: Owner
+      required: false
+    serial_number:
+      title: Serial Number
       type: string
-      description: Environment ID
+      description: Serial number
       required: false
     sha1:
+      title: SHA1
       type: string
       description: SHA1
       required: false
+    valid_from:
+      title: Valid From
+      type: string
+      description: Valid from
+      required: false
+    valid_until:
+      title: Valid Until
+      type: string
+      description: Valid until
+      required: false
+  extracted_files:
+    av_label:
+      title: Available Label
+      type: string
+      description: Available label
+      required: false
+    av_matched:
+      title: Available Matched
+      type: integer
+      description: Available matched
+      required: false
+    av_total:
+      title: Available Total
+      type: integer
+      description: Available total
+      required: false
+    description:
+      title: Description
+      type: string
+      description: Description
+      required: false
+    file_available_to_download:
+      title: File Available To Download
+      type: boolean
+      description: File available to download
+      required: false
+    file_path:
+      title: File Path
+      type: string
+      description: File path
+      required: false
+    file_size:
+      title: File Size
+      type: integer
+      description: File size
+      required: false
     md5:
+      title: MD5
       type: string
       description: MD5
       required: false
+    name:
+      title: Name
+      type: string
+      description: Name
+      required: false
+    runtime_process:
+      title: Runtime Process
+      type: string
+      description: Runtime process
+      required: false
+    sha1:
+      title: SHA1
+      type: string
+      description: SHA1
+      required: false
+    sha256:
+      title: SHA256
+      type: string
+      description: SHA256
+      required: false
+    threat_level:
+      title: Threat Level
+      type: integer
+      description: Threat level
```

Comment:
Is there any range for the threat level.

Reply:
API do not give any information about that